cost-analyzer/values.yaml

```diff
@@ -26,6 +26,14 @@ global:
   podAnnotations: {}
     # iam.amazonaws.com/role: role-arn
 
+pricingCsv:
```

Comment:
would it help to make this more generic? e.g. `customPricingProvider`

Reply:
Think the specific is preferred? customPricingProvider is an implementation detail.